src/components/overlays/OverlayController.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import Alert from './Alert';
+import Confirm from './Confirm';
+
+const OverlayController = () => (
+    <>
+        <Alert />
+        <Confirm />
+    </>
+);
+
+export default OverlayController;
```

Comment:
Hmm, I would prefer to say
        <Alert />
        <Confirm />
at App.tsx. Because this class is just a dummy wrapper. Maybe it could work if it had a better name? ModalContainer?

Reply:
OverlayContainer